docs/tutorials/embedded/wine_detector.md

```diff
@@ -100,13 +101,14 @@ with open('synset.txt', 'r') as f:
     synsets = [l.rstrip() for l in f]
 
 # Load the network parameters
-sym, arg_params, aux_params = mx.model.load_checkpoint('Inception_BN', 0)
+sym, arg_params, aux_params = mx.model.load_checkpoint('Inception-BN', 126)
```

Comment:
magic number?

Reply:
It is the number of epoch.

Reply:
Right, what I mean is this probably shouldn't be hard-coded?

Reply:
Agree.

Reply:
Maybe, but if you follow the instruction, the model donloaded has that epoch for the checkpoint. What do you suggest instead?